BDArmory/GameModes/Asteroids.cs

```diff
@@ -781,6 +781,11 @@ IEnumerator Float()
                             if (weaponManager == null) continue;
                             offset = weaponManager.vessel.transform.position - asteroids[i].transform.position;
                             factor = 1f - (float)offset.sqrMagnitude / 1e6f; // 1-(r/1000)^2 attraction. I.e., asteroids within 1km.
+                            if (BDArmorySettings.RUNWAY_PROJECT && BDArmorySettings.RUNWAY_PROJECT_ROUND == 69) // Punish immobile turrets
```

Comment:
`RUNWAY_PROJECT_ROUND == 69` is the waypoints round. Did you mean `RUNWAY_PROJECT_ROUND == 70` for S6R10?

The PR comments mention
> Adjust anomalous attraction factor for asteroids in orbit. Attraction now gradually increases from 1km, peaking at 500m, then gradually decreases to 0 at 0km to avoid asteroids accelerating quickly into nearby vessels.

but I don't see this anywhere. In any case, I think there still needs to be some attraction when the asteroids are close. We could change `factor` to something like the following (which avoids calculating `offset.magnitude`)
```csharp
var R = (float)offset.sqrMagnitude / 1e6f;
factor = 0.25f + 3f * R * (1f - R); // Attraction: 0.25 at 0m, 1 at 707m, 0 at 1.038km
```
![factor](https://github.com/user-attachments/assets/b4936e3c-8c8f-4ec7-908f-5e8d0dc143ef)